src/Gtk/Perspex.Gtk/GtkPlatform.cs

```diff
@@ -10,6 +10,18 @@
 using Perspex.Platform;
 using Perspex.Shared.PlatformSupport;
 
+namespace Perspex
+{
+	public static class GtkApplicationExtensions
+	{
+		public static AppT UseGtkSubsystem<AppT>(this AppT app) where AppT : Application
```

Comment:
I think we perhaps need to work out our naming here. It seems you're using the word "subsystem" for the windowing system but not the rendering system? To me they are both equal, so they should probably have the same nomenclature... "Subsystem" is probably redundant, how about just `UseGtk` the same as you have `UseCairo`?

Or was the intention to make it clear that you had to pick one windowing system and one rendering system? In that case maybe `UseGtkWindowing` and `UseCairoRenderer` or something? Not sure...